WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -389,7 +391,7 @@ private extension QuickStartTourGuide {
     }
 
     func dismissCurrentNotice() {
-        ActionDispatcher.dispatch(NoticeAction.dismiss)
+        ActionDispatcher.dispatch(NoticeAction.clearWithTag(noticeTag))
         ActionDispatcher.dispatch(NoticeAction.empty)
```

Comment:
I'm curious whether we still need this. 🤔 But I think we should leave it for now cause it's not a good idea to cause a regression. It doesn't look like `QuickStartTourGuide` has unit tests so that makes things more difficult.

Reply:
I don't know this feature deeply enough to make that call. It might be superfluous. I'll add a comment before merging.